Cargo.toml

```diff
@@ -26,15 +26,22 @@ wasmtime-wasi = { git = "https://github.com/bytecodealliance/wasmtime.git", rev
 wasi-common = { git = "https://github.com/bytecodealliance/wasmtime.git", rev = "8ac4bd1d0d8228b97a88b1841cfc0247e9ef4306", default-features = false }
 env_logger = "0.7"
 log = "0.4"
+tokio = { version = "0.2", features = ["macros"] }
+warp = { version = "0.2", features = ["tls"] }
+
+serde = { version = "1.0", features = ["derive"] }
+serde_yaml = "0.8"
+serde_json = "1.0"
+serde_derive = "1.0"
+openssl = { version = "0.10", features = ["vendored"] }
+reqwest = { version = "0.10.7", features = ["blocking", "json", "native-tls"] }
```

Comment:
`reqwest` appears only to be used bin a test binary. The test binary should be built as a test and then this can become a `dev-dependency` rather than forcing it to be used for all builds.

Reply:
https://doc.rust-lang.org/rust-by-example/testing.html

You probably want integration tests. But unit tests are also valuable.